pkg/reconciler/v1alpha1/autoscaling/hpa/hpa_test.go

```diff
@@ -53,7 +54,7 @@ func TestControllerCanReconcile(t *testing.T) {
 
 	scaleClient := &fakescaleclient.FakeScaleClient{Fake: kubeClient.Fake}
 	scaleClient.PrependReactor("get", "deployments", func(action ktesting.Action) (bool, runtime.Object, error) {
-		return true, scaleResource(testNamespace, testRevision, withLabelSelector("a=b")), nil
+		return true, scaleResource(testNamespace, testRevision), nil
```

Comment:
probably scaleResource is no longer necessary?

Reply:
There is logic in both PA's tests still using it (like this actually).

My fix was just for the scaleClient setup for the table test, which we can consider generalizing for this as well.

Reply:
Rather what I meant is that I added this to pass selector to SKS, it wasn't there before, so now that the SKS takes in objref instead this particular scale resource is probably no longer required.